NetflixRouletteWebpack/src/components/counter/Counter.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import "./Counter.scss";
+
+class Counter extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { value: 0 };
+  }
+
+  changeValue(increase) {
+    const { value } = this.state;
+    this.setState(increase ? { value: value + 1 } : { value: value - 1 });
```

Comment:
@Marysosh You can simplify like `value++` and `value--`

Reply:
I tried and it breaks the functionality, the counter stops working. Moreover, eslint throws an exception:
<img width="883" alt="image" src="https://user-images.githubusercontent.com/98745170/198544141-50c5f23d-5775-4425-91ad-1e5d8493db71.png">